routes/mongo.js

```diff
@@ -0,0 +1,34 @@
+require('dotenv').config();
+var express = require('express');
+var router = express.Router();
+const MongoClient = require('mongodb').MongoClient;
+const uri = "URIstring"
```

Comment:
Lets still include this string `const uri = mongodb://${process.env.DBUSER}:${process.env.DBPASSWORD}@${process.env.DBHOST}/${process.env.DB};`
here